examples/TicTacToe-Console.links

```diff
@@ -1,9 +1,8 @@
 #tictactoe console edition
 
 typename Player = [|X|O|];
-typename Option(a) = [|None|Some:a|];
 
-var initialState = [None, None, None, None, None, None, None, None, None] : [Option(Player)];
+var initialState = [Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing] : [Maybe(Player)];
```

Comment:
I think this change is fine, but it worth noting that it is not really necessary. There might be reasons to use an `Option` type over `Maybe` type in examples, e.g. an ML crowd will be familiar with `Option`, whereas a Haskell crowd would be mostly familiar with `Maybe`.

Reply:
I think it is worthwhile for consistency with the prelude and to avoid having to re-declare Option whenever needed(i.e. keep things DRY).